From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Mon, 9 Jul 2018 23:50:28 +0200
Subject: base-files: fix feed list in PKG_CONFIG_DEPENDS

FEEDS_ENABLED and FEEDS_DISABLED are derived from FEEDS_AVAILABLE, not
FEEDS_INSTALLED.

Signed-off-by: Matthias Schiffer <mschiffer@universe-factory.net>

diff --git a/package/base-files/Makefile b/package/base-files/Makefile
index 7175daec3a0a1a3ecd08ceee4d06e2c0eae4b29a..ef29798d5de855e35d178cb02f356bb3e77a9bd0 100644
--- a/package/base-files/Makefile
+++ b/package/base-files/Makefile
@@ -27,7 +27,7 @@ PKG_CONFIG_DEPENDS += \
 	CONFIG_PER_FEED_REPO \
 	CONFIG_PER_FEED_REPO_ADD_DISABLED \
 	CONFIG_PER_FEED_REPO_ADD_COMMENTED \
-	$(foreach feed,$(FEEDS_INSTALLED),CONFIG_FEED_$(feed))
+	$(foreach feed,$(FEEDS_AVAILABLE),CONFIG_FEED_$(feed))
 
 include $(INCLUDE_DIR)/package.mk
 
